src/unity/schemas/IdentityUser.yml

```diff
@@ -0,0 +1,34 @@
+properties:
+  idpeId:
+    type: string
+    description: user's id in idpe
+  email:
+    type: string
+    description: user's email
+  firstName:
+    type: string
+    nullable: true
+    description: user's first name
+  lastName:
+    type: string
+    nullable: true
+    description: user's last name
+  operatorRole:
+    type: string
+    nullable: true
+    enum:
+      - read-only
+      - read-write
+    description: role permission whether this operator is "read-only" | "read-write"
+  accountCount:
+    type: integer
+    description: number of accounts the user belongs to
+  orgCount:
+    type: integer
+    description: number of orgs the user belongs to within their active account
+required:
+  - id
+  - idpeId
```

Comment:
This field list doesn't match with what's above; we need to remove either `id` or `idpeId` from here depending on how we decide to name IDs.